resources/hotels.py

```diff
@@ -0,0 +1,96 @@
+from flask import abort
+from flask_restful import Resource, reqparse, fields, marshal
+
+# TODO: create a postgres database and store these there
+hotels = [
+    {
+    'id': 1,
+    'hotel_name': 'The Asteroid',
+    'total_double_rooms': 10,
+    'total_queen_rooms': 10,
+    'total_king_rooms': 10,
+    'created_date': '2020-06-03'
+    }
+]
+
+hotel_fields = {
+    'id': fields.Integer,
+    'hotel_name': fields.String,
+    'total_double_rooms': fields.Integer,
+    'total_queen_rooms': fields.Integer,
+    'total_king_rooms': fields.Integer,
+    'created_date' : fields.String,
+    'uri': fields.Url('hotel')
+}
+
+class HotelList(Resource):
+    def __init__(self):
+        self.reqparse = reqparse.RequestParser() # these lines are for input validation
+        self.reqparse.add_argument('hotel_name', type = str, required = True,
+            help = "No hotel name provided.", location = 'json')
+        self.reqparse.add_argument('total_double_rooms', type = int, required = True,
+            help = "Total number of double rooms in the hotel not provided.", location = 'json')
+        self.reqparse.add_argument('total_queen_rooms', type = int, required = True,
+            help = "Total number of queen rooms in the hotel not provided.", location = 'json')
+        self.reqparse.add_argument('total_king_rooms', type = int, required = True,
+            help = "Total number of king rooms in the hotel not provided.", location = 'json')
```

Comment:
it is necessary to require these 3 arguments? Can we assume that it's 0 if we don't supply it?